### PR TITLE
Renamed "r" to "rad", since var "r" is used as "red" on line 77

### PR DESCRIPTION
**Pull request by arthurjdam**
Only a legibility and log tweak, not essential for functionality.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ var equirectToCubemapFaces = (function() {
 					var rC = srgbToLinear(inData[pC+0]|0), gC = srgbToLinear(inData[pC+1]|0), bC = srgbToLinear(inData[pC+2]|0), aC = (inData[pC+3]|0)*(1.0 / 255.0);
 					var rD = srgbToLinear(inData[pD+0]|0), gD = srgbToLinear(inData[pD+1]|0), bD = srgbToLinear(inData[pD+2]|0), aD = (inData[pD+3]|0)*(1.0 / 255.0);
 
-					var r = (rA*(1.0-mu)*(1.0-nu) + rB*mu*(1.0-nu) + rC*(1.0-mu)*nu + rD*mu*nu);
+ 					var r = (rA*(1.0-mu)*(1.0-nu) + rB*mu*(1.0-nu) + rC*(1.0-mu)*nu + rD*mu*nu);
```
Comment (thomcc) [inline index.js]: nit: inserted leading whitespace.

**Review by thomcc — CHANGES_REQUESTED** [review]
Also a nice catch. Will merge after nit is addressed.

**Comment by arthurjdam** [comment]
Good one. I wonder what my editor was thinking

**Comment by thomcc** [comment]
lg2m

**Pull request merged**